pkg/system/prlimit.go

```diff
@@ -68,19 +69,58 @@ func SetCPUTimeLimit(pid int, value uint64) error {
 	return limiter.SetCPUTimeLimit(pid, value)
 }
 
+// QEMU progress report uses carriage returns for the percentages, the default split function
```

Comment:
Comment should start with `// ScanLinesWithCR`

Reply:
done